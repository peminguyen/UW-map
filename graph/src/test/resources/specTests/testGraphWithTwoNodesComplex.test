CreateGraph graphWithTwoNodesComplex

AddNode graphWithTwoNodesComplex A
AddNode graphWithTwoNodesComplex B

AddEdge graphWithTwoNodesComplex A A eAA
AddEdge graphWithTwoNodesComplex A B eAB
AddEdge graphWithTwoNodesComplex B A eBA
AddEdge graphWithTwoNodesComplex B B eBB

ListNodes graphWithTwoNodesComplex
ListChildren graphWithTwoNodesComplex A
ListChildren graphWithTwoNodesComplex B