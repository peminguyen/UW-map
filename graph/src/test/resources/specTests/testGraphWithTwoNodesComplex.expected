created graph graphWithTwoNodesComplex

added node A to graphWithTwoNodesComplex
added node B to graphWithTwoNodesComplex

added edge eAA from A to A in graphWithTwoNodesComplex
added edge eAB from A to B in graphWithTwoNodesComplex
added edge eBA from B to A in graphWithTwoNodesComplex
added edge eBB from B to B in graphWithTwoNodesComplex

graphWithTwoNodesComplex contains: A B
the children of A in graphWithTwoNodesComplex are: A(eAA) B(eAB)
the children of B in graphWithTwoNodesComplex are: A(eBA) B(eBB)